tests/Integration/SimpleFileFetcherTest.php

```diff
@@ -31,27 +31,27 @@ public function testGetThisFileFromGitHub() {
 			'https://raw.githubusercontent.com/JeroenDeDauw/FileFetcher/master/tests/Integration/SimpleFileFetcherTest.php'
 		);
 
-		$this->assertInternalType( 'string', $contents );
+		$this->assertIsString( $contents );
 
-		$this->assertInternalType( 'integer', strpos( $contents, __FUNCTION__ ) );
+		$this->assertContains( __FUNCTION__, $contents );
 	}
 
 	public function testGivenNotFoundFile_exceptionIsThrown() {
 		$fetcher = new SimpleFileFetcher();
+		$invalidRemoteFileUrl = 'http://raw.github.com/JeroenDeDauw/FileFetcher/master/foo.php';
 
 		$this->expectException( FileFetchingException::class );
-		$fetcher->fetchFile(
-			'http://raw.github.com/JeroenDeDauw/FileFetcher/master/foo.php'
-		);
+		$this->expectExceptionMessage( 'Could not fetch file: ' . $invalidRemoteFileUrl );
+		$fetcher->fetchFile( $invalidRemoteFileUrl );
 	}
 
 	public function testGivenInvalidUrl_exceptionIsThrown() {
 		$fetcher = new SimpleFileFetcher();
+		$invalidRemoteFileUrl = 'foo bar baz';
 
 		$this->expectException( FileFetchingException::class );
-		$fetcher->fetchFile(
-			'foo bar baz'
-		);
+		$this->expectExceptionMessage( 'Could not fetch file: ' . $invalidRemoteFileUrl );
```

Comment:
I'm not so sure about dynamically constructing the expected message here

Reply:
In this test, this expected exception message will be current, not dynamic.

Just check the exception message is thrown as actual.

Do you mean using the `expectExceptionMessageRegExp ` to handle the dynamic constructor?

Reply:
I mean compared to this

    		$this->expectExceptionMessage( 'Could not fetch file: foo bar baz' );

There is value in not having the test implementation mirror the production code

Reply:
@JeroenDeDauw, I see that.

I think we should let them dynamically because of consistency.

Reply:
Consistency with what?

Reply:
I don't think it matters much here btw, it's just a small difference. My comments here are more an FYI

Reply:
I misunderstand what you mean.

And what about your suggestion about this?